mypyc/lib-rt/getargs.c

```diff
@@ -374,8 +374,6 @@ vgetargskeywords(PyObject *args, PyObject *kwargs, const char *format,
             int res = PyDict_GetItemStringRef(kwargs, kwlist[i], &current_arg);
             if (res == 1) {
```

Comment:
```suggestion
            if (unlikely(res == 1)) {
```
To fully restore the behavior prior to #17930.